python/ray/serve/tests/test_cli.py

```diff
@@ -478,5 +478,41 @@ def test_idempotence_after_controller_death(ray_start_stop, use_command: bool):
     ray.shutdown()
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="File path incorrect on Windows.")
+@pytest.mark.parametrize("config_update", [True, False])
+def test_deploy_config_update(ray_start_stop, config_update: bool):
+    config_file_name = os.path.join(
+        os.path.dirname(__file__), "test_config_files", "pid.yaml"
+    )
+    subprocess.check_output(["serve", "deploy", config_file_name])
+
+    ray.init(address="auto", namespace=SERVE_NAMESPACE)
```

Comment:
I think you can remove the `ray.init()` and `serve.start()` calls (and their corresponding `shutdown()` calls). You don't seem to be interacting with Ray through the Python API here. The `ray_start_stop` parameter in this test's signature should trigger the `ray_start_stop` fixture, which should handle starting and stopping Ray.